### ISSUE TITLE
"Microcephaly" should belong to Condition domain, not to the Observation

### ISSUE THREAD

**Issue by dimshitc** [OHDSI/Vocabulary-v5.0#726]
[Microcephaly](https://athena.ohdsi.org/search-terms/terms/606878) should belong to Condition domain, not to the Observation (I understand the peaks are all about Observation in this case, so the exclusion should be added)

**Comment by cgreich** [comment]
@dimshitc:

That's debatable. As such, the meaning is just "too big head" and "too small head". It's not any specific diagnosis. 

What is the use case?

**Comment by dimshitc** [comment]
this is the source code mapped to this one
http://www.icd9data.com/2014/Volume1/740-759/742/742.1.htm
see Approximate Synonyms
Brain hypoplasia, congenital
Congenital brain hypoplasia
Congenital hypoplasia of brain
Hydromicrocephaly
Microcephaly
Primary microcephaly

so it´s not just a "too small head" but some brain hypoplasia.

No use case, I just reviewing the mapping changes affecting our CDMs. But people who work on on these conditions will be definitely confused.

**Comment by dimshitc** [comment]
Well, isn't it kinda late to fix the error in a vocabulary when there's a use case already?
Use case driven approach works very well when developing new logic, but not when fixing things, I think.

**Comment by cgreich** [comment]
It's the other way around: Only when there is a use case the vocab needs fixing. Otherwise it is in vain.

But in this case the use case is not a use case: Just because some ICD9CM says something is a synonym it isn't. In fact, you are not expecting the above list to be synonymous (they are not, they are just telling coders what to put into that particular ICD9CM), you are using it as an argument that all these are conditions.

And they are. Except microcephaly. Again, they probably mean "microcephaly of unknown diagnosis". Which doesn't make it a condition. The "unknown" is the condition. The microcephaly is a "Finding of head circumference", almost a measurment.

SNOMED also has another idea: [Microcephaly](https://athena.ohdsi.org/search-terms/terms/40321920) has an "Inactive possibly_equivalent_to active" relationship to [Congenital microcephaly](https://athena.ohdsi.org/search-terms/terms/606879). Maybe you use that?

**Comment by dimshitc** [comment]
Yes, Maybe here's a problem with mapping as well: ICD9 I'm referring to should be mapped to [Congenital microcephaly](https://athena.ohdsi.org/search-terms/terms/606879). And [Congenital microcephaly](https://athena.ohdsi.org/search-terms/terms/606879) should change it's domain to Condition.

**Comment by dimshitc** [comment]
Are you going to make these changes in the next release:
ICD9 I'm referring to should be mapped to [Congenital microcephaly](https://athena.ohdsi.org/search-terms/terms/606879). And [Congenital microcephaly](https://athena.ohdsi.org/search-terms/terms/606879) should change it's domain to Condition.
?
@TinyRickC137 @Alexdavv

**Comment by TinyRickC137** [comment]
With all respect, most likely it will be done during the summer release

**Comment by m-khitrun** [comment]
Hi, @dimshitc. The mapping of Microcephaly was improved: https://athena.ohdsi.org/search-terms/terms/35210698. Thank you!
